control.lua

```diff
@@ -324,26 +390,43 @@ function print_game(...)
   game.print(text)
 end
 
+function print_file(...)
```

Comment:
Is it possible to merge this function with `print_game` ?

Reply:
Sure, we can combine the similar logic so it is only written once.